scripts/jest/testschema.graphql

```diff
@@ -8,6 +8,7 @@ type Root {
   nodes(ids: [ID!]): [Node]
   username(name: String!): Actor
   usernames(names: [String!]!): [Actor]
+  usertemp(tuple: tupleUser!): Actor
```

Comment:
Basically, we need to add support for three new types of identifying argument: **arrays**, **numbers**, and **objects**. How about this for the sample schema:

```graphql
checkinSearchQuery(query: CheckinSearchInput!): CheckinSearchResult
fastestRoute(waypoints: [Waypoint!]!): Route
task(number: Int!): Task
```